build.gradle.kts

```diff
@@ -25,7 +25,7 @@ if (!File("$rootDir/.git").exists()) {
 }
 
 group = "net.onelitefeather"
-version = "1.4.0"
+version = "1.0.0"
```

Comment:
Don't downgrade the version to 1.0.0. Because we focus so far it possible to use the schema. SemVer.org

Reply:
sorry I was testing the update thing and forgot to change it back, will fix